packages/logs/src/boot/startLogs.ts

```diff
@@ -63,30 +64,37 @@ export function doStartLogs(
   }
 
   function reportError(error: RawError) {
-    errorLogger.error(
-      error.message,
-      combine(
-        {
-          date: error.startClocks.timeStamp,
-          error: {
-            kind: error.type,
-            origin: error.source,
-            stack: error.stack,
-          },
-        },
-        error.resource
-          ? {
-              http: {
-                method: error.resource.method,
-                status_code: error.resource.statusCode,
-                url: error.resource.url,
-              },
-            }
-          : undefined
-      )
-    )
+    const messageContext: Partial<LogsEvent> = {
+      date: error.startClocks.timeStamp,
+      error: {
+        kind: error.type,
+        origin: error.source,
+        stack: error.stack,
+      },
+    }
+    if (error.resource) {
+      messageContext.http = {
+        method: error.resource.method as any, // TODO: Set method to upper case (it's a breaking change),
+        status_code: error.resource.statusCode,
+        url: error.resource.url,
+      }
+    }
+    logger.error(error.message, messageContext)
   }
+
+  function reportConsoleLog(log: ConsoleLog) {
+    const messageContext: Partial<LogsEvent> = {}
+    if (log.status === StatusType.error) {
+      messageContext.error = {
+        origin: log.source,
+        stack: log.stack,
+      }
+    }
+    logger.log(log.message, messageContext, log.status)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the message context is not really a LogsEvent... I think it could be simpler as:
```suggestion
    const messageContext
    if (log.status === StatusType.error) {
      messageContext = {
        error: {
          origin: log.source,
          stack: log.stack,
        }
      }
    }
    logger.log(log.message, messageContext, log.status)
```

Reply:
Makes sense, ok!